tests/setup-tests.js

```diff
@@ -2,4 +2,8 @@ import enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import 'isomorphic-fetch';
 
+jest.mock('react-native-localize', () => ({
```

Comment:
How about moving this mock to the `__mocks__` directory as we do for `react-native-config`?

Example:
https://github.com/rootstrap/react-native-base/blob/master/__mocks__/react-native-config.js

Docs:
https://jestjs.io/docs/en/manual-mocks#mocking-node-modules

Reply:
Yeah, seems better. I'll change it

Reply:
@fsarachu Done, thanks for the feedback!